src/main/java/se/sundsvall/casedata/api/model/Address.java

```diff
@@ -1,20 +1,16 @@
 package se.sundsvall.casedata.api.model;
 
-import jakarta.validation.constraints.Size;
-
-import se.sundsvall.casedata.integration.db.model.enums.AddressCategory;
-
 import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.Size;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
-import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
+import se.sundsvall.casedata.integration.db.model.enums.AddressCategory;
```

Comment:
On the subject of API-fixes. Many fields in this have `@Size(max = 255)` should probably be reflected in the api with `maxLenght = 255`

Reply:
This is probably valid for other classes too